client/plots/DEanalysis.js

```diff
@@ -1,4 +1,4 @@
-import * as client from '../src/client'
+import { axisstyle, make_table_2col } from '../src/client'
```

Comment:
recommend to replace  make_table_2col  with `import { table2col } from '#dom/table2col'`